docs/jwst.rst

```diff
@@ -0,0 +1,254 @@
+.. _jwst_instruments:
+
+
+*****************************
+JWST Instrument Model Details
+*****************************
+
+
+The following describes specific details for the various JWST instrument classes. See also :ref:`the references page <references>` for information on data sources. 
+
+
+One general note is that the webbpsf class interfaces do not attempt to exactly
+model the implementation details of all instrument mechanisms, particularly for
+NIRCam and NIRISS that each have multiple wheels. In webbpsf, the
+``filter`` attribute of a given class is used to select any and all filters,
+even if as a practical matter a given filter is physically installed in a
+"pupil" wheel instead of a "filter" wheel. Likewise any masks that affect the
+aperture shape are selected via the ``pupil_mask`` attribute even if physically
+an optic is in a so-called "filter" wheel.
+
+
+Each SI has a ``detector`` attribute that can be used to select among its
```

Comment:
The `<blah>_list` attributes bear mentioning somewhere, maybe just after this. I was thinking of `detector_list`, but the same naming scheme works for filters and masks.